frontend/packages/ux-editor/src/components/leftMenu/LeftMenu.module.css

```diff
@@ -34,3 +34,31 @@
 .receipt {
   border-bottom: 1px solid #c9c9c9;
 }
+
+.layoutSetList {
+  border-color: #E4E7EA;
+  border-style: solid;
+  border-width: 1px 0;
+  max-height: 10%;
+  overflow-y: auto;
+  resize: vertical;
+  margin-left: 20px;
+  padding: 10px;
+}
+
+.configureLayoutSetButton {
+  text-align: left;
+}
+
+.configureLayoutSet {
+  display: flex;
+  margin: 10px;
+}
+
+.configureLayoutSetInfo {
+  justify-content: left;
+  max-width: 500px;
+  border-radius: 20px;
+  padding: 10px;
+  line-height: 1.6;
+}
```

Comment:
Du kan godt lage en egen CSS-fil for dette, siden det brukes i en egen komponent.